nbproject/project.properties

```diff
@@ -6,8 +6,8 @@ app.name=${branding.token}
 ### if left unset, version will default to today's date
 app.version=4.12.0
 ### build.type must be one of: DEVELOPMENT, RELEASE
-build.type=RELEASE
-#build.type=DEVELOPMENT
+#build.type=RELEASE
+build.type=DEVELOPMENT
```

Comment:
This change needs to be removed. We need this branch to be using the DEVELOPMENT build.type.

Reply:
done